src/main/java/org/jenkinsci/test/acceptance/po/ArtifactManagement.java

```diff
@@ -46,8 +46,14 @@ public ArtifactManagement(JenkinsConfig config) {
 
     public void clear() {
         try {
-            while (true) {
+            while (control("artifactManagerFactories/repeatable-delete").exists()) {
                 control("artifactManagerFactories/repeatable-delete").click();
+                // We allow slow Javascripts to remove the deleted element before we click again (JENKINS-32569)
+                try {
+                    Thread.sleep(500);
+                } catch (InterruptedException ex) {
+                    // just ignore and continue the cycle
```

Comment:
I'm not sure it's really safe to catch it like this since an InterruptedException usually indicates something is actively trying to stop the thread from running anymore and tries to be nice about it, you should be nice back and exit gracefully.